cfme/tests/services/test_catalog_item.py

```diff
@@ -37,24 +41,42 @@ def dialog():
     flash.assert_success_message('Dialog "{}" was added'.format(dialog))
     yield dialog
 
+    try:
```

Comment:
I'd say I like yield fixture's teardown more than callbacks in tests. :+1: . nice test improvement.